aries-site/src/examples/components/menu/MenuHeaderExample.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Header, Menu } from 'grommet';
+import { AppIdentity } from 'aries-core';
+
+export const MenuHeaderExample = () => {
+  const items = [
+    { label: 'Change username' },
+    { label: 'Reset Password' },
+    { label: 'Logout' },
+  ];
+
+  return (
+    <Header background="background-contrast" pad="small" fill>
```

Comment:
Do we want a maxWidth on Header? On wide screens the example gets stretched out pretty far.

Reply:
I'm not sure I'm following -- do you mean in the context that someone were to copy and paste to use or in our site? 

I feel like the header width show be drive by the layout, in the sense that it can take up as much space as it can but the layout should be the one to restrict content width. Let me know your thoughts. This would be guidance we'd want to have in the site somewhere about creating proper layouts

Reply:
Sorry. I was meaning put a Box around the Header to limit its maxwidth in the example. When things get too far spaced out, it can be difficult to track a row from beginning to end -- more relevant in the list example where if the row is wide it becomes difficult to know which user I am affecting / intending to affect. 

Soft opinion/suggestion.

![Screen Shot 2020-04-24 at 11 02 42 AM](https://user-images.githubusercontent.com/1756948/80238284-6847db00-861b-11ea-8f0b-a63bf53857c4.png)
![Screen Shot 2020-04-24 at 11 02 52 AM](https://user-images.githubusercontent.com/1756948/80238297-6da52580-861b-11ea-8336-3545216b15a1.png)

Reply:
I like it for the List example -- I'll update that one then merge.